sys/netlink/netlink_message_parser.h

```diff
@@ -46,7 +46,7 @@ struct linear_buffer {
 static inline void *
 lb_alloc(struct linear_buffer *lb, int len)
 {
-	len = roundup2(len, sizeof(uint64_t));
```

Comment:
You missed the __aligned on the struct for this one

Reply:
Humm, for some reason I assumed it shared the struct instead of redefining it.  Fixed.